en/community/mailing-lists/ruby-talk-guidelines/index.md

```diff
@@ -9,7 +9,9 @@ You should follow these guidelines when posting to the ruby-talk mailing list.
 
 
 1. **Always** be friendly, considerate, tactful, and tasteful. We want to
-   keep this list hospitable and safe for everyone.
+   keep this list hospitable to the growing ranks of newbies, very
+   young people, and their teachers, as well as cater to fire breathing
```

Comment:
s/their/the/, at least. :)

Reply:
Thanks, but this part is just a revert :D. Do you think I should amend it? I think it may go to a separate PR...

Reply:
"s/their/the" usage makes reading more difficult for non natives english readers and for reading software (used by people with vision problems)